src/lib/markbind/src/lib/markdown-it-shared/markdown-it-icons.js

```diff
@@ -1,15 +1,22 @@
 const octicons = require('@primer/octicons');
 
 module.exports = require('markdown-it-regexp')(
-    /:(fa[brs]|glyphicon|octicon)-([a-z-]+):/,
+    /:(fa[brs]|glyphicon|octicon|octiconlight)-([a-z-]+)~?([a-z-]+)?:/,
```

Comment:
how about `:(fa[brs]|glyphicon|octicon|octiconlight)-([a-z-]+)(~[a-z-]+)?:` so that `?` won't have to be repeated twice?

Reply:
Realised `match[3]` needs to be updated as well here; Probably better to revert it to the original